README.md

```diff
@@ -48,3 +48,15 @@ development environment you can use.
    ```bash
    docker-compose run --rm obs-docu daps -vv -d DC-obs-all html
    ```
+
+## Update documentation
+
+The main steps to update the documentation are:
+
+- Fork [obs-docu](https://github.com/openSUSE/obs-docu) project and clone the forked one in your computer.
```

Comment:
I believe it would read better like this:

[...] and clone the fork on your computer.